spec/amp-managing-user-state.md

```diff
@@ -361,6 +361,18 @@ Our approach will take advantage of two types of [AMP variable substitutions](./
 <a href="https://example.com/step2.html?ref_id=CLIENT_ID(uid)" data-amp-replace="CLIENT_ID">
 ```
 
+**Alternative solution for passing Client ID to the outgoing links:** Define the new query parameter `ref_id` as part of the data attribute `data-amp-addparams` and for queries that needs parameter substitution provide those details as part of `data-amp-replace`. With this approach the URL would look clean and the parameters specified on `data-amp-addparams` will be dynamically added
```

Comment:
@rudygalfi - As you own this doc and are more familiar with the content, can you review that this is accurate and the correct location for it?